### PR TITLE
[RFR][1LP] Fixed automate schema crud

### PR DESCRIPTION
**Pull request by quarckster**
{{pytest: -v cfme/tests/automate/test_class.py::test_schema_crud}}

### CODE REVIEW

**Comment by jawatts** [comment]
:+1:

File: cfme/automate/explorer/klass.py [hunk 1]
```diff
@@ -371,8 +371,11 @@ def row_id(self):
             max_time = Input(name=ParametrizedString('fields_max_time_{@row_id}'))
 
             def delete(self):
-                self.browser.click(
-                    './/img[@alt="Click to delete this field from schema"]', parent=self)
+                if self.browser.product_version < '5.10':
```
Comment (jawatts) [inline cfme/automate/explorer/klass.py]: Question:
Why not use VersionPick here?

Comment (quarckster) [reply cfme/automate/explorer/klass.py]: VersionPick is a descriptor and it's supposed to be used for class attributes. For view instances `self.browser.product_version` should be used.

Comment (psav) [reply cfme/automate/explorer/klass.py]: Yeh VersionPick could be used here, though it is generally thought to be more efficient to do a simple comparison here.

**Pull request merged**